packages/embark-ui/src/actions/index.js

```diff
@@ -360,6 +360,13 @@ export const web3EstimateGas = {
   failure: (error, payload) => action(WEB3_ESTIMAGE_GAS[FAILURE], {web3Error: error, contract: payload.contract})
 };
 
+export const WEB3_IS_DEPLOYED = createRequestTypes('WEB3_IS_DEPLOYED');
```

Comment:
Would it be possible to rename this just `IS_DEPLOYED` or `IS_CONTRACT_DEPLOYED`? Because it's not really web3(js) dependent.

Reply:
In this case, it is actually web3-dependent, due to the `services/web3.js` that provides the underlying functionality for deployment, gas estimation, and checking if a contract is deployed. Maybe once we abstract that out somehow, we can change the name of all the actions, sagas, props, etc.